### ISSUE TITLE
Locked online editing bug

### ISSUE THREAD

**Issue by IceflowRE** [red-eclipse/base#37]
If you get on a server which does editing and is locked. And you dont have the rights to edit and you go in edit and scroll fast to change something, you get a message error and disconnect from server.
This error should be caught and should not lead to the disconnection from the server.

**Comment by ZeroKnight** [comment]
Could you post the error message? Try to reproduce the error with different levels of `verbose`. If you have admin on the server, do the same with `serverdebug`.

**Comment by IceflowRE** [comment]
2015-02-17 16:17.52 [msg error] from: 1, cur: 44, msg: -1, prev: -1

You mean something like this? :D

**Comment by qreeves** [comment]
Yeah, that's a message not allowed error. The checktype command is failing the client for sending an editing message outside of edit mode. This is a client side desync/lag thing, because they switch to edit mode before sending the message, which the server then corrects after the fact if they're not allowed to be in edit mode. We'd have to come up with a way of only switching after the server has permitted it (or have the client check better if they're allowed to in the first place).
